src/pretty_set.cpp

```diff
@@ -78,11 +78,40 @@ bool check_reservoir(Reservoir& reservoir, Model<bool>* seen, vector<ArrayCoordi
 	return true;
 }
 
+bool check_TN_reservoir(Reservoir& reservoir, Model<bool>* seen, vector<ArrayCoordinate>* used_points, BigModel& big_model) {
+
+	Model<short>* DEM = big_model.DEM;
+	vector<ArrayCoordinate> temp_used_points;
+	ArrayCoordinate centre_point = convert_coordinates(convert_coordinates(reservoir.pour_point), DEM->get_origin());
+
+	for (int x = -6; x <= 5; x++) {
+		for (int y = -6; y <= 5; y++) {
+			if (ceil(sqrt(pow(x+0.5, 2) + pow(y+0.5, 2))) <= 5) {
+				if (seen->get(centre_point.row + y, centre_point.col + x))
+					return false;
```

Comment:
More magic numbers - should these be configurable? Will they change with the size of the system? 300m*300m doesn't feel big enough for a 1500GWh system. This will also create elliptical turkeys nests when away from the equator which probably isn't ideal.